protos/local/core_interface.proto

```diff
@@ -115,9 +115,15 @@ message SignalWorkflow {
     temporal.api.history.v1.WorkflowExecutionSignaledEventAttributes signal = 1;
 }
 
+message CancelActivity {
+    string activity_id = 1;
```

Comment:
I don't think we need activity_id here as we already have a task token at the 'Task' level.